* Fabian Groffen <grobian@gentoo.org>
  We don't want Fink stuff, we only want our own stuff.  No need to
  patch configure.in, as upstream doesn't like this patch, as they put
  it in on purpose.

--- configure
+++ configure
@@ -2384,12 +2384,6 @@
     opsys=darwin
     # Define CPP as follows to make autoconf work correctly.
     CPP="${CC-cc} -E -no-cpp-precomp"
-    # Use fink packages if available.
-    if test -d /sw/include && test -d /sw/lib; then
-      GCC_TEST_OPTIONS="-I/sw/include -L/sw/lib"
-      CPP="${CPP} ${GCC_TEST_OPTIONS}"
-      NON_GCC_TEST_OPTIONS=${GCC_TEST_OPTIONS}
-    fi
   ;;
 
   ## AT&T 3b2, 3b5, 3b15, 3b20
